src/utils/fxa.js

```diff
@@ -88,13 +123,12 @@ async function getSubscriptions(bearerToken) {
         Authorization: `Bearer ${bearerToken}`
       }
     })
+    const resp = getResp.json()
+    if (!getResp.ok) throw resp;
 
-    if (!getResp.ok) {
-      throw new InternalServerError(`bad response: ${getResp.status}`)
-    } else {
-      console.info(`get_fxa_subscriptions: success`)
-      return await getResp.json()
-    }
+    console.info(`get_fxa_subscriptions: success`)
```

Comment:
💯 FYI I filed a ticket in backlog to find and fix all of these (we have a ton of call sites in old JS code that look like this), but I am in favor of fixing when it's in the way too.